### PR TITLE
Add prettier and linter

### PR DESCRIPTION
**Pull request by eyalgolan**
added:
* prettier
* eslint-config-airbnb

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,26 +1,32 @@
...
+### Credits
 
 ### License
+
 Inside this repo we are using the following open-source projects:
+
 1. [Node.js](https://github.com/nodejs/node)
 2. [react](https://github.com/facebook/react)
 3. [mongoose](https://github.com/Automattic/mongoose)
 4. [octonode](https://github.com/pksunkara/octonode)
```
Comment (teddy4445) [inline README.md:20]: add "eslint" license here

File: src/server/data/index-data.json [hunk 2]
```diff
@@ -1 +1,9 @@
-{"biography":"654","featuredPublications":[{"name":"3","description":"21","link":"32","type":"13"}],"currentProjects":[{"name":"21","description":"51","link":"5"}],"name":"321","age":"32"}
\ No newline at end of file
+{
```
Comment (teddy4445) [inline src/server/data/index-data.json:1]: update file according to: https://github.com/teddy4445/lecture_website_template/blob/develop/data/jsons/index.json

File: src/server/server.js [hunk 3]
```diff
@@ -1,46 +1,51 @@
...
-import mailgun from 'mailgun-js' 
-import { generateDbConnectionFailedEmail } from './src/resources/emails'
-import path from 'path'
-import express from 'express'; 
-import cors from 'cors'
-import mongoose from 'mongoose'
-import dotenv from 'dotenv'
-import routers from './src/routes/'
+import User from "./src/models/user";
+import mailgun from "mailgun-js";
```
Comment (teddy4445) [inline src/server/server.js:2]: add "mailgun" license into the README file

**Review by teddy4445 — APPROVED** [review]
Just the small readme fixes and fine for merge

File: README.md (hunk 1 above)
Comment (eyalgolan) [reply README.md:20]: done

File: src/server/server.js (hunk 3 above)
Comment (eyalgolan) [reply src/server/server.js:2]: done

File: src/server/data/index-data.json (hunk 2 above)
Comment (eyalgolan) [reply src/server/data/index-data.json:1]: updated, please verify

**Pull request merged**